### PR TITLE
Feat: Completed Homepage and added basic routes

### PR DESCRIPTION
**Pull request by vinc3nati**
- Completed Home Page UI
image

- Added basic Routes

### CODE REVIEW

**Review by rohit8686 — APPROVED** [review]
Everything looks good and i like that you're changing titles for different pages

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,59 @@
...
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+section {
+  padding: 2.5rem 1.8rem;
 }
 
-.App-header {
-  background-color: #22201c;
```
Comment (Rohanmond) [inline src/App.css:17]: hard coded color

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      </div>
+      <div className="user">
+        <Link to="/auth">
+          <button className="btn icon-btn">
+            <FaUserCircle />
+            Login
+          </button>
+        </Link>
+
+        {/* <span className="text-tertiary">username</span>
```
Comment (Rohanmond) [inline src/components/navbar/Navbar.jsx:42]: delete comments here

**Review by Rohanmond — APPROVED** [review]

File: src/components/navbar/Navbar.jsx (hunk 2 above)
Comment (vinc3nati) [reply src/components/navbar/Navbar.jsx:42]: okay

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,25 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import { Footer } from "./components/footer/Footer";
+import { Navbar } from "./components/navbar/Navbar";
+import { Home } from "./pages/Home/Home";
```
Comment (PeeyushPrashant) [inline src/App.js:5]: try giving the route as navbar/Navbar.js, as it makes more sense, try implementing in other routes as well

**Review by PeeyushPrashant — APPROVED** [review]
the code structure looks very clean

**Pull request merged**